apps/sms/services/README.md

```diff
@@ -0,0 +1,91 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+There are 3 layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Services define high-level semantics to access the model. They are the only
```

Comment:
Can you please can pick simple example to describe every layer to have common ground and understanding  (e.g. loading of threads in InboxView)?

It's a bit unclear to me what layer uses ```bridge.client``` and ```bridge.service``` and where "manager's" responsibilities end and "service" responsibilities begin?